### DISCUSSION TITLE
library/

### DISCUSSION THREAD

**Discussion by giscus** [Air-duino/document#22] [comment]
# library/

支持的外设 目前，AirMCU 通过 Arduino API 支持以下外设。 外设Air001Air32F103Air401备注 :---::---::---::---: ADC✅🔨🔨| DAC❌🔨🔨| GPIO✅✅✅| I2C✅🔨🔨| Servo✅🔨🔨| SPI✅🔨🔨| PWM✅🔨🔨| UART✅🔨✅| USB❌🔨❌| ✅...

https://arduino.luatos.com/library/

**Comment by waitter-01** [comment]
Arduino平台怎么使用air001的看门狗呀
